Add IntesisHome climate platform (#8917)
home-assistant/home-assistant.github.io: source/_components/climate.intesishome.markdown

---
layout: page
title: "IntesisHome AC Controller"
description: "Instructions how to integrate IntesisHome AC devices with Home Assistant"
date: 2019-03-12 00:00
sidebar: true
comments: false
sharing: true
footer: true
logo: intesishome.png
ha_category: Climate
ha_release: 0.96

> ha_iot_class: "Cloud Push"

Reviewer: ```suggestion
ha_iot_class: Cloud Push
```

Note: IntesisHome products are a separate product line to IntesisBox. This platform does not support IntesisBox devices (which can be controlled locally using the WMP protocol).

To set it up, add the following information to your `configuration.yaml` file:

```yaml
climate:
  - platform: intesishome
    username: YOUR_USERNAME
    password: YOUR_PASSWORD
```

{% configuration %}
username:
  description: Your username for [IntesisHome.com](https://user.intesishome.com) 
  required: true
  type: string
password:
  description: Your password for IntesisHome
  required: true
  type: string
{% endconfiguration %}

This component opens a TCP connection with the IntesisHome API to receive temperature and status updates, and to issue commands.
By default the component will be named using the friendly device name from the IntesisHome website or application.
If internet connectivity is lost the device will be marked as unavailable after 5 minutes.

### {% linkable_title Supported services %}

> Available services:

Reviewer: What you could do is add a description to the services, it makes it a little clearer to the users what they are for 😉

---
layout: page
title: "IntesisHome AC Controller"
description: "Instructions how to integrate IntesisHome AC devices with Home Assistant"
date: 2019-03-12 00:00
sidebar: true
comments: false
sharing: true
footer: true

> logo: intesishome.png

Reviewer 2: I cannot find this file, did you forget to add it?

Author: Sorry missed that step! Added now, cheers

ha_category: Climate

> ha_release: 

Reviewer 2: Please fill in a HA release. I suggest to keep it at `0.96` for now.

ha_iot_class: "Cloud Push"
---

> The `IntesisHome` climate platform let you control [IntesisHome](https://www.intesishome.com) devices via the IntesisHome cloud. IntesisHome provide integrations with air conditioners including including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.

Reviewer 2: ```suggestion
The `IntesisHome` climate platform lets you control [IntesisHome](https://www.intesishome.com) devices via the IntesisHome cloud. IntesisHome provides integrations with air conditioners including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
```

Note: IntesisHome products are a separate product line to IntesisBox. This platform does not support IntesisBox devices (which can be controlled locally using the WMP protocol).

To set it up, add the following information to your `configuration.yaml` file:

```yaml
climate:
  - platform: intesishome
    username: YOUR_USERNAME
    password: YOUR_PASSWORD
```

{% configuration %}
username:
  description: Your username for IntesisHome
  required: true
  type: string
password:
  description: Your password for IntesisHome
  required: true
  type: string
{% endconfiguration %}

> - **username** (*Required*): Username of an account authorised to control your IntesisHome device on [IntesisHome.com](https://user.intesishome.com).

Reviewer 2: Not sure why this is here? Please use the `description` of the above documentation configuration tags.

- **password** (*Required*): Password for the IntesisHome account.

This component opens a TCP connection with the IntesisHome API to receive temperature and status updates, and to issue commands.
By default the component will be named using the friendly device name from the IntesisHome website or application.
If internet connectivity is lost the device will be marked as unavailable after 5 minutes.

> ### {% linkable_title Supported services %}

Reviewer 2: ```suggestion
### {% linkable_title Supported services %}

```

> Available services: 

Reviewer 2: ```suggestion
Available services:

```

---
layout: page
title: "IntesisHome AC Controller"
description: "Instructions how to integrate IntesisHome AC devices with Home Assistant"
date: 2019-03-12 00:00
sidebar: true
comments: false
sharing: true
footer: true
logo: intesishome.png
ha_category: Climate
ha_release: 
ha_iot_class: "Cloud Push"
---

The `IntesisHome` climate platform let you control [IntesisHome](https://www.intesishome.com) devices. IntesisHome provide integrations with air conditioners including including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.

To set it up, add the following information to your `configuration.yaml` file:

```yaml
climate:
  - platform: intesishome
    username: YOUR_USERNAME

>     password: !secret intesishome_password

Reviewer: ```suggestion
    password: YOUR_PASSWORD
```

```

The password must be stored in your `secrets.yaml` file:
```yaml

> intesishome_password: YOUR_PASSWORD

Reviewer: I think we can skip this, put your password in the secrets.yam filel is not required, but something users can do optional.

Author: Thanks, have updated that now

Reviewer: I must say that I started a discussion about it on Discord, because it would not be wrong to put the passwords in the `secrets.yaml` file and in the way you had previously drafted. Only at the moment it is not the "standard" that we use, who knows this may change in the future 😉

```

{% configuration %}
username:
  description: Your username for IntesisHome
  required: true
  type: string
password:

>   description: Your 

Reviewer: ```suggestion
  description: Your password for IntesisHome
```
